.github/workflows/platformDeploy.yml

```
@@ -266,3 +266,25 @@ jobs:
     needs: [android, desktop, iOS, web]
     steps:
       - run: echo "Status android = ${{ needs.android.result }}, desktop = ${{ needs.desktop.result }}, iOS = ${{ needs.iOS.result }}, web = ${{ needs.web.result }}"
+
+      - name: Set version
+        run: echo "VERSION=$(npm run print-version --silent)" >> $GITHUB_ENV
+
+      - name: Get Release Pull Request List
+        id: getReleasePRList
+        uses: Expensify/Expensify.cash/.github/actions/getReleasePullRequestList@main
+        with:
+          TAG: ${{ env.VERSION }}
```

NAB but are we sure this doesn't work?

```suggestion
          TAG: ${{ $(npm run print-version --silent) }}
```